### PR TITLE
Lesson 9 - more about function

### PR DESCRIPTION
**Pull request by MrTim**
Lesson content for lesson 9.
Covered here: arguments, return and callback function(as a general idea)

Still in progress: claswork.

### CODE REVIEW

**Comment by BTBTravis** [comment]
So far so good. I like the bit about callbacks hopefully they will more easily understand addEventListener now

**Review by BTBTravis — APPROVED** [review]

File: lessons/9_more_about_functions/index.html [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+				<ul class="fragment">
+					<li>Speed and time always the same</li>
+					<li>We can only alert the distance</li>
+				</ul>
+			</section>
+			<section>
+				<h3>But we can improve!</h3>
+			</section>
+			<section>
+				<h3>Function arguments</h3>
```
Comment (tobiastimm) [inline lessons/9_more_about_functions/index.html]:
```suggestion
				<h3>Function parameters</h3>
```

File: lessons/9_more_about_functions/index.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+				<p class="fragment"><b>Arguments</b> are like variable boxes which you can use inside function</p>
```
Comment (tobiastimm) [inline lessons/9_more_about_functions/index.html]:
```suggestion
				<p class="fragment"><b>Parameters</b> are like variable boxes which you can use inside function</p>
```
There is a difference between arguments and parameters. 
A parameter is a variable in a function definition
An argument  is the data you pass into the parameters when you call the function

File: lessons/9_more_about_functions/index.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+				<h5 class="fragment">Syntax</h5>
+				<pre class="fragment">
+					<code>
+function yourFunctionName(argument1, argument2) {
```
Comment (tobiastimm) [inline lessons/9_more_about_functions/index.html]:
```suggestion
function yourFunctionName(parameter1, parameter2) {
```

File: lessons/9_more_about_functions/index.html [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+console.log(`In console: ${distance}`);
+
+// or in another example
+alert(calculateDistance(60, 2) + calculateDistance(200, 2))
+						</code>
+					</pre>
+				</div>
+			</section>
+			<section>
+				<h3>We can pass a <b>function</b> as argument</h3>
```
Comment (tobiastimm) [inline lessons/9_more_about_functions/index.html]:
```suggestion
				<h3>We can pass a <b>function</b> as an argument</h3>
```

File: lessons/9_more_about_functions/index.html (hunk 2 above)
Comment (MrTim) [reply lessons/9_more_about_functions/index.html]: True! good catch

**Pull request merged**